build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+    id 'java-library'
+}
+
+subprojects {
+    apply plugin: 'java-library'
+}
+
+allprojects {
+    group 'org.quartz-scheduler'
+    version '2.4.0-SNAPSHOT'
+
+    java.sourceCompatibility(JavaVersion.VERSION_11)
```

Comment:
This should probably either be done with toolchains, or with a release setting on the relevant compiler tasks: https://docs.gradle.org/current/userguide/toolchains.html